.github/workflows/dev-layer-gmp.yml

```
@@ -1,23 +1,23 @@
 #
---- L9_META ---
+# --- L9_META ---
 l9_schema: 1
```

actionlint (1.7.11)

[error] unexpected key "l9_schema" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)